frame/lending/src/lib.rs

```diff
@@ -846,41 +857,98 @@ pub mod pallet {
 			Ok(should_warn)
 		}
 
+		/// Initiate liquidation of individual position for particular borrower within mentioned
+		/// market. Returns 'Ok(())' in the case of successful initiation, 'Err(DispatchError)' in
+		/// the opposite case.
+		/// - `liquidator` : Liquidator's account id.
+		/// - `market_pair` : Index and configuration of the market from which tokens were borrowed.
+		/// - `account` : Borrower's account id whose debt are going to be liquidated.
+		fn liquidate_position(
+			liquidator: &<Self as DeFiEngine>::AccountId,
+			market_pair: &(&<Self as Lending>::MarketId, MarketConfigOf<T>),
+			borrow_asset: <T as DeFiComposableConfig>::MayBeAssetId,
+			account: &<Self as DeFiEngine>::AccountId,
+		) -> Result<(), DispatchError> {
+			let (market_id, market) = market_pair;
+			if Self::should_liquidate(market_id, account)? {
+				let collateral_to_liquidate = Self::collateral_of_account(market_id, account)?;
+
+				let source_target_account = Self::account_id(market_id);
+
+				let unit_price =
+					T::Oracle::get_ratio(CurrencyPair::new(market.collateral_asset, borrow_asset))?;
+
+				let sell = Sell::new(
+					market.collateral_asset,
+					borrow_asset,
+					collateral_to_liquidate,
+					unit_price,
+				);
+				T::Liquidation::liquidate(
+					&source_target_account,
+					sell,
+					market.liquidators.clone(),
+				)?;
+				// TODO: @mikolaichuk: Justify why is it fine to move borrow rent
```

Comment:
as for dicussion is happend to slack, what we decided? i assume it is safe

Reply:
Agree